### PR TITLE
Add link to job board on careers page

### LINKED ISSUE #269: Add link to Discord on careers page
Add link to Discord on careers page, pointing out that there are several job ads in the #job-board channel on the UF Discord server.

### PR DESCRIPTION
**Pull request by nathanshammah**
Close #269. It provides more resources for folks looking for job opportunities even when no job openings at UF are available.

### CODE REVIEW

**Review by natestemen — CHANGES_REQUESTED** [review]
The spacing looks kind of off to me.
Screen Shot 2022-11-02 at 9 16 41 AM

I think there should be a double break between paragraphs, and the period should come before the first linebreak after the added material.

**Comment by natestemen** [comment]
Can we also change "there are no currently open positions" to "there are currently no open positions."?

File: careers.html [hunk 1]
```diff
@@ -77,6 +77,11 @@ <h3> There are no currently open positions</h3>
               <a href="mailto:info@unitary.fund">info@unitary.fund</a>.
               <br />
 
+              A continuously updated list of work opportunities is available on the <a href="http://discord.unitary.fund">Unitary Fund Discord server</a>, in the <br><a href="https://discord.gg/eQ6C6ZpM">#job-board channel</a></br>.
+              <br />
+                          
+              https://discord.gg/eQ6C6ZpM
+
```
Comment (nathanshammah) [inline careers.html:84]:
```suggestion
              A continuously updated list of work opportunities is available on the <a href="http://discord.unitary.fund">Unitary Fund Discord server</a>, in the <b><a href="https://discord.gg/eQ6C6ZpM">#job-board channel</a></b>.
              <br />
                          
          
```

**Comment by nathanshammah** [comment]
Thanks @natestemen.

**Review by natestemen — COMMENTED** [review]
Do you think it's worth specifying the jobs on #job-board are not necessarily UF jobs?

**Comment by nathanshammah** [comment]
@natestemen good point. I also implement the other feedback now

**Review by natestemen — APPROVED** [review]

**Pull request merged**